client/src/components/charts/DashMap.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { PropTypes } from 'react';
+import { Map, Marker, Popup, TileLayer } from 'react-leaflet';
+import * as chart from '../../utilities/chart';
+
+require('../../../node_modules/leaflet/dist/leaflet.css');
+require('../../styles/DashMap.scss');
+
+
+export default function DashMap({ visualisation, datasets }) {
+  const chartData = chart.getChartData(visualisation, datasets);
+
+  return (
+    <div className="DashMap dashChart">
+      <Map
+        center={[0, 0]}
+        zoom={2}
+        scrollWheelZoom={false}
```

Comment:
This is perhaps a matter of personal taste, but I think scroll-to-zoom has very bad usability on embedded maps unless the map is the only feature on the page - otherwise, a user scrolls halfway down the page, then suddenly the embedded map hooks into the scroll events and the map starts zooming instead of the page, in a manner that's unintuitive and surprising.

Given that both the map creation dialog and the dashboards themselves are complex pages with several elements, and the user may reasonably expect to scroll up and down the page in the usual fashion, I am a strong advocate for disabling scroll-to-zoom.